src/rest/Query.ts

```diff
@@ -238,4 +238,9 @@ export interface IQuery {
    * Specifies an array of request to retrieve facet values for the MLFacet component
    */
   facets?: IFacetRequest[];
+
+  /**
+   * Indicates whether the facet values should be returned in their current order.
+   */
+  freezeFacetOrder?: Boolean;
```

Comment:
Speaking with Suh-Shin, the api team has decided to move the `freezeFacetOrder` property into the facetOptions object (I think the equivalent for us would be the `IFacetRequest`).